timesketch/api/v1/resources.py

```diff
@@ -25,6 +25,8 @@
 POST /sketches/:sketch_id/event/
 POST /sketches/:sketch_id/event/annotate/
 POST /sketches/:sketch_id/views/
+
+POST /tsctl/searchindices/
```

Comment:
this can be just /searchindices/

Thinking about this, we should create 2 exnpoints here:
/index/ - Create or get ES index (create using es.create_index)
/searchindices/ - Create TS index mapping (the one you implemented here)